### DISCUSSION TITLE
What mean the compiler switch OPENSSL_DEV_NO_ATOMICS

### DISCUSSION THREAD

**Discussion by UlrichKamke** [openssl/openssl#23413] [General Discussion]
Hello all,

it seams the -DOPENSSL_DEV_NO_ATOMICS can be a work around for my compiler issue with ICC on Windows.
What does the OPENSSL_DEV_NO_ATOMICS mean and  what does it do?

Best regards 
Ulrich

**Comment by mattcaswell** [comment]
It turns off detection of compiler atomics support. Not having this is likely to have significant performance impacts.
